src/pegin/components/create/PegInForm.vue

```diff
@@ -100,6 +107,8 @@ export default defineComponent({
     const environmentContext = EnvironmentContextProviderService.getEnvironmentContext();
     const typeSummary = TxStatusType.PEGIN;
     const orientationSummary = TxSummaryOrientation.VERTICAL;
+    const liqualityWarningMessage = 'Some users have encountered diffilcuties while attempting to create new Bitcoin accounts in Liquality. It\'s important to note that this issue is unrelated to the 2WP app. if you experience this problem, we recommend selecting another wallet.';
```

Comment:
There is a typo here.
```suggestion
    const liqualityWarningMessage = 'Some users have encountered difficulties while attempting to create new Bitcoin accounts in Liquality. It\'s important to note that this issue is unrelated to the 2WP app. if you experience this problem, we recommend selecting another wallet.';
```